src/database/entities/notes.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UserEntity } from './user.entity';
+
+@Entity({ name: 'notes' })
+export class NotesEntity {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({ default: 'Title !' })
+    title: string;
+
+    @Column({ default: 'Write something here! ' })
+    body: string;
+
+    @Column({ type: 'timestamp', default: () => 'NOW()' })
+    createdAt?: Date;
+
+    @Column({ default: 0 })
+    timezoneOffset: number;
```

Comment:
I'm not sure if it is important. What if we store date in some standard form (some ISO?) and then on frontend get time zone and convert it to proper zone?

Reply:
I think we don't have to store timezone difference, we can convert it "on the go"